Source/WebGPU/WGSL/TypeCheck.cpp

```diff
@@ -660,7 +681,7 @@ void TypeChecker::visit(AST::CallExpression& call)
             return;
         }
 
-        typeError(target.span(), "Cannot call value of type: '", **targetType, "'");
+        typeError(target.span(), "cannot call value of type '", *targetBinding->type, "'");
```

Comment:
A complete aside: I watched a linguistic professor talk on how non-native speakers use "cannot" instead of "can't".